### PR TITLE
AM-3004 : Add hearing-viewer & hearing-manager to CTSC users for SSCS

### PR DESCRIPTION
**Pull request by prashanthkumarnb**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/AM-3004

### Change description ###
This PR is Dependent  on RAS hmcts/am-role-assignment-service#2091

> NB: This PR has temporary changes to Jenkins and Chart file configuration that need reversing before merger.

### CODE REVIEW

File: Jenkinsfile_CNP
Comment (mattnayler) [inline Jenkinsfile_CNP:79]: NB: This temporary change must be reverted before merging.

File: charts/am-org-role-mapping-service/values.preview.template.yaml
Comment (mattnayler) [inline charts/am-org-role-mapping-service/values.preview.template.yaml:38]: NB: This temporary change must be reverted before merging.

**Review by mattnayler — DISMISSED** [review]
LGTM

**Review by mikebrownccd — DISMISSED** [review]
Still needs to have the temp url reverted before merging into Master

File: Jenkinsfile_CNP
Comment (mikebrownccd) [reply Jenkinsfile_CNP]: Reverted

File: charts/am-org-role-mapping-service/values.preview.template.yaml
Comment (mikebrownccd) [reply charts/am-org-role-mapping-service/values.preview.template.yaml]: Reverted

**Review by mikebrownccd — APPROVED** [review]
LGTM

**Review by mattnayler — APPROVED** [review]
Re-approved after preview config change 2dfd7f76a63f0d87f2fa6d1b01bc9f5b71f27ea1.

**Pull request merged**
